node/core/approval-voting/src/lib.rs

```diff
@@ -1366,8 +1375,19 @@ async fn handle_approved_ancestor(
 		},
 	);
 
+	// `reverse()` to obtain the ascending order from highest to lowest
+	// block within the canddidates, which is the expected order
```

Comment:
how come spellcheck did not catch this?
```suggestion
	// block within the candidates, which is the expected order
```

Reply:
It's a dev comment `//` (not `///`) and those are not checked with the current CI flags. We'd need to add `--dev` to make that happen.

Reply:
I see, seems like a good idea to me to check dev comments as well.